content/concepts/listings/how-the-listing-search-works/index.mdx

```diff
@@ -137,6 +287,11 @@ from the user's service provider entity. You can then use the default
 listing search to query the service provider profiles with all the query
 options described above.
 
+This article in our Help Center has more detailed instructions on the
+option of using listings as user profiles:
+
+- [How to configure listings to look like user profiles](https://www.sharetribe.com/help/en/articles/12953405-how-to-configure-listings-to-look-like-user-profiles)
+
 ### Custom user search endpoint in the template server
 
 The Sharetribe Integration API does have an endpoint for querying users.
```

Comment:
Not a part of the changes you made but I think we should replace "trusted context" from this paragraph, and replace it with "secure environment"